Add macOS and Debian install (#28)
lena-larionova/docs.konghq.com: app/gateway/2.6.x/install-and-run/macos.md

---

> id: page-install-method
> title: Install Kong Gateway on macOS
> header_title: macOS Installation
> header_icon: /assets/images/icons/icn-installation.svg
> breadcrumbs:
>   Installation: /install
> ---

Reviewer: ```suggestion
title: Install Kong Gateway on macOS
badge: oss
---
```

> 

Reviewer: ```suggestion
<!-- Banner with links to latest downloads -->
<!-- The install-link and install-listing-link classes are used for tracking, do not remove -->

{:.install-banner}
> See the MacOS
> [**Homebrew formula**]({{ site.repos.homebrew }}){:.install-listing-link}
>
> (latest {{site.base_gateway}} version: {{site.data.kong_latest_ce.version}})
```

## Prerequisites

You have a supported system with root or [root-equivalent](/gateway/{{page.kong_version}}/plan-and-deploy/kong-user) access.

> ## Packages
> 
> - [Homebrew Formula]({{ site.repos.homebrew }})
> 

Reviewer: ```suggestion
```
Moved into banner.

> 1. **Install Kong**

Reviewer: ```suggestion
## Download and install

1. Install {{site.base_gateway}}:
```

    Use the [Homebrew](https://brew.sh/) package manager to add Kong as a tap and install it:

    ```bash
    brew tap kong/kong
    brew install kong
    ```

> 2. **Prepare your database or declarative configuration file**

Reviewer: ```suggestion
2. Prepare your database or declarative configuration file:
```

    Then, as root, add `kong.conf.default` to `/etc`:

    ```bash
    sudo mkdir -p /etc/kong
    sudo cp kong.conf.default /etc/kong/kong.conf
    ```

    **Note for Kong < 0.15**: with Kong versions below 0.15 (up to 0.14), use
    the `up` sub-command instead of `bootstrap`. Also note that with Kong <
    0.15, migrations should never be run concurrently; only one Kong node
    should be performing migrations at a time. This limitation is lifted for
    Kong 0.15, 1.0, and above.

    **Without a database**

    If you are going to run Kong in [DB-less mode](/gateway-oss/latest/db-less-and-declarative-config/),
    you should start by generating a declarative config file. The following command will generate a `kong.yml`
    file in your current folder. It contains instructions about how to populate it.

    ``` bash
    kong config init
    ```

    After populating the `kong.yml` file, edit your `kong.conf` file. Set the `database` option
    to `off` and the `declarative_config` option to the path of your `kong.yml` file:

    ``` conf
    database = off
    declarative_config = /path/to/kong.yml
    ```

> 3. **Start Kong**

Reviewer: ```suggestion
3. Start {{site.base_gateway}}:
```

    ```bash
    kong start [-c /path/to/kong.conf]
    ```

> 4. **Use Kong**
> 
>     Verify that Kong is running:

Reviewer: ```suggestion
4.  Verify that {{site.base_gateway}} is running:
```
